chat/chat.js

```diff
@@ -0,0 +1,32 @@
+const messageContainer = document.querySelector("#message-container")
```

Comment:
It works fine, but for better realization - I'd like to provide something similar to global state (as I see it, it could be object with 2 pairs of key and list of message objects as value)

In this case, you will have opportunity to create new logic, just by passing new properties for message object (that could be stored in global state)